### DISCUSSION TITLE
Could not poll the database

### DISCUSSION THREAD

**Discussion by douwei-2** [firefly-iii/firefly-iii#9562] [Firefly III]
Hi,
Created a docker and getting this error. not sure what is going on. I have looked at all the other issues related to this and not sure if any of them apply.

Could not poll the database: SQLSTATE[HY000] [1045] Access denied for user 'firefly'@'192.168.48.2' (using password: YES) (Connection: mysql, SQL: select `id`, `name`, `data` from `configuration` where `name` = is_demo_site and `configuration`.`deleted_at` is null limit 1)

**Comment by JC5** [comment]
Hi, thanks for starting a discussion. There are plenty of threads that discuss this issue. Most often, the database is initialized with a different password from what Firefly III is using to connect.

**Comment by douwei-2** [comment]
Hi James,
Yes, and I apologize. I have tried eveything and still no go. I have a
compose file that works on another system but not mine. I really want to
use firefly internally in my home network and get rid of quicken. But most
guides out there are for reverse proxy and all that..and just dont need it.
Im also a noob. AGain sorry for making a post that was answered elsewhere.

Doug

**Reply by shadowjustice** [reply]
Have you tried deleting all the DB files?
